### ISSUE TITLE
Show charts

### ISSUE THREAD

**Issue by manwar** [manwar/theweeklychallenge#141]
Have a place in the new design for the charts that we have in current website. It doesn't have to be exactly same format but would like to keep the same chart. Please do share if you have better ideas.

https://theweeklychallenge.org/chart/

**Comment by kanonkartik** [comment]
Hello @manwar, as an Outreachy applicant, Could you please confirm if contributions are still open? I'm excited about the opportunity to make valuable contributions to this project.

**Comment by manwar** [comment]
Yes, if I am not mistaken, the contribution period is still on. You are most welcome to this project. @manwar

**Comment by kanonkartik** [comment]
Yes!!Thank you for the confirmation! @manwar  I'm thrilled about the opportunity to contribute to this project. However, I encountered an issue trying to join the Slack project's public chat. Could you please assist me with joining the Slack channel? Looking forward to collaborating with everyone!

**Comment by manwar** [comment]
@kanonkartik please share your email and I will send you the invite.

**Comment by kanonkartik** [comment]
my email is:- kart00ik@gmail.com
thankyou @manwar

**Comment by manwar** [comment]
Just sent you invitation. @kanonkartik

**Comment by himanshisrestha** [comment]
@manwar ,  I would like to work on this issue please assign this to me .
